databricks_cli/stack/api.py

```diff
@@ -91,6 +91,29 @@ def deploy(self, config_path, **kwargs):
         click.echo("Saving stack status to {}".format(status_path))
         self._save_json(status_path, new_stack_status)
 
+    def download(self, config_path, **kwargs):
+        """
+        Downloads a stack given stack JSON configuration template at path config_path.
+
+        Loads the JSON template as well as status JSON if stack has been deployed before.
+        Changes working directory to the same directory as where the config file is, then
+        calls on download_config to do the stack downloading.
```

Comment:
The docstr here have too much implementation details, which may change in the future. The documentation here should make sense when user use "--help" to see them.